### PR TITLE
upgrade.php and UTF8 scripts fixes

### PR DESCRIPTION
**Pull request by stefaweb**
This change is [Reviewable](https://reviewable.io/reviews/mailwatch/1.2.0/361)

### CODE REVIEW

**Comment by stefaweb** [comment]
Added mtalog_ids missing from utf8 sql script.

File: upgrade.php [hunk 1]
```diff
@@ -484,5 +484,6 @@ function getTableIndexes($table)
     echo "*** ERROR/WARNING SUMMARY ***\n";
     foreach ($errors as $error) {
         echo $error . "\n";
+        echo "\n";
```
Comment (endelwar) [inline upgrade.php]: Why this extra \n for each error row?

Comment (stefaweb) [inline upgrade.php]: May be not well placed.
The intend was to add a blank line at the end when we launch the script in console.

Comment (stefaweb) [inline upgrade.php]:
```
*** ERROR/WARNING SUMMARY ***
MailScanner.conf: HighScoringSpamActions != store (=delete)

antispam-dev:/var/www/html/mailscanner#
```

File: upgrade.php [hunk 2]
```diff
@@ -31,8 +31,8 @@
  */
 
 header("Content-type: text/plain\n\n");
-//require '/var/www/html/mailscanner/functions.php';
-require __DIR__ . '/mailscanner/functions.php';
+require '/var/www/html/mailscanner/functions.php';
+//require __DIR__ . '/functions.php';
```
Comment (endelwar) [inline upgrade.php]: `/mailscanner` is missing from commented require

**Comment by stefaweb** [comment]
I have a new batch of localization to PR. I wait for this one before.

**Comment by endelwar** [comment]
done!
remember to sync your fork before issuing new PR (http://docs.mailwatch.org/dev/contributing.html#contributing-to-mailwatch)

**Pull request merged**